### PR TITLE
chore: deploy non-snapshot when building from verta/main

### PR DESCRIPTION
**Pull request by liam-verta**
Tested locally by switching to `install` instead of deploy. 
I'm going to try one other change for testing.

### CODE REVIEW

File: verta_build_and_publish.sh [hunk 1]
```diff
@@ -3,18 +3,31 @@
...
+    echo The "revision" property in pom.xml must end with "-SNAPSHOT" for this script to work correctly.
+    echo Actual value: revision = $PROJECT_REVISION
+    exit 1
+fi
+
+# Insert branch name in project version
+export PROJECT_VERSION=${PROJECT_REVISION/%-SNAPSHOT/-${VERSION_SUFFIX}-SNAPSHOT}
+
+# When building verta main replace -SNAPSHOT with commit info
+if [ "$BRANCH_NAME" == "ln/deploy-main" ]; then
```
Comment (conradoverta) [inline verta_build_and_publish.sh]: Old?

**Review by coryverta — APPROVED** [review]

File: verta_build_and_publish.sh (hunk 1 above)
Comment (liam-verta) [reply verta_build_and_publish.sh]: Nope. New for testing. :) 
Reverted already.

**Comment by liam-verta** [comment]
This jenkins run will show the non-snapshot behavior using this branch.  https://jenkins.dev.verta.ai/job/build/job/external/job/druid/45/console

This is not a big deal because we do not auto-update `java-services` with new builds from this project. 

When the builds for this PR completes, I'll file a PR showing what updating the dependency in `java-services` looks like. 

NOTE: this build must complete before pushing an update over there, even for just PR purposes. The `java-services` build depends on the output of `distribution ` in this project which is the last output of this build.

**Comment by liam-verta** [comment]
https://github.com/VertaAI/java-services/pull/189 shows what the update will look like in `java-services`.

**Pull request merged**